src/core/cluster.py

```diff
@@ -465,15 +466,17 @@ def _broker_status(self) -> Status:
         if not self.runs_broker:
             return Status.ACTIVE
 
-        if not self.zookeeper:
-            return Status.ZK_NOT_RELATED
+        # FIXME
```

Comment:
**suggestion**: maybe we could rename `_broker_status` to `_broker_with_zk_status` or something like that, and add a KRaft equivalent. We would then choose which one to use in `ready_to_start`